### ISSUE TITLE
Add resource section in the documentation

### ISSUE THREAD

**Issue by joelostblom** [altair-viz/altair#2414]
What do you think about adding a section in the docs with links to and short descriptions of learning resources and packages related to / building on Altair?

I went through all Github repos mentioning Altair that was updated since Aug 2020 and labeled as either a Python and Notebook repo. From that search, the resource section could look something like this:

---

# Resources

## Learning material

- [Jupyter Notebooks tutorials and examples](https://github.com/altair-viz/altair_notebooks)
- [A data visualization curriculum from the UW data group](https://uwdata.github.io/visualization-curriculum)
- [Altair tutorial given at PyCon 2018](https://altair-viz.github.io/altair-tutorial)

## Related packages

- [Vega-Lite - The higher-level visualization grammar that Altair implements in Python](https://vega.github.io/vega-lite/)
- [altair_saver - Enables saving charts to a variety of output types](https://github.com/altair-viz/altair_saver)
- [altair_data_server - Data transformer plugin that transparently serves data for charts](https://github.com/altair-viz/altair_data_server)
- [altair_pandas - Altair backend for the pandas plotting API](https://github.com/altair-viz/altair_pandas)
- [vega_datasets -  Offline access to the Vega datasets used in the Altair documentation](https://github.com/altair-viz/vega_datasets)
- [nx_altair - Draw interactive NetworkX graphs with Altair](https://github.com/Zsailer/nx_altair)
- [gif - Create animated Altair gifs via loops](https://github.com/maxhumber/gif)
- [Vegawidget - R interface to Altair](https://vegawidget.github.io/altair/)

---

For full disclosure, I am working on a data visualization course using Altair at the University of British Columbia and a wrapper package for making a few EDA plots quicker with Altair. When they are further along, I will suggest adding them here, but also happy if this stays as is and just makes the info easier to find.

**Comment by joelostblom** [comment]
It could be nice with some elaboration on the learning resources, I remember it being difficult to figure out what to read when I started learning Altair. In the end I mostly used the documentation and the UW data curriculum which I think made for a great combo, but I'm not familiar with the other two I linked above.

I just built [a Jupyter Book for the UW data visualization](https://uwdata.github.io/visualization-curriculum/intro.html) and I would be happy to build one for the `altair_notebooks` repo as well, but I saw https://github.com/altair-viz/altair_notebooks/issues/23, so maybe I should wait until the material is brought up to date?
